app/models/claims/claim_window.rb

```diff
@@ -51,6 +51,10 @@ def self.current
     find_by_date(Date.current)
   end
 
+  def self.previous
```

Comment:
Should we add a spec for this?

Reply:
Yup. They're upcoming. I'll try to finish them up by end of today.